src/components/modes/teacher/TeacherView.js

```diff
@@ -152,7 +83,8 @@ export class TeacherView extends Component {
   });
 
   state = {
-    selectedStudent: null,
+    fromDate: moment.utc('04-01-2019'),
```

Comment:
Do we not want this to be somewhat dynamic? Based at least on today's date or on the data.

Reply:
True, I wasn't sure which approach to take. Will default it to the boundaries of data for now and we can change it later.